sevntu-checks/.ci/eclipse-compiler-javac.sh

```diff
@@ -30,7 +29,10 @@ if [ ! -f $ECJ_PATH ]; then
     cd ..
 fi
 
-wget https://github.com/checkstyle/checkstyle/blob/checkstyle-$2/config/org.eclipse.jdt.core.prefs
+wget https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-$2/config/org.eclipse.jdt.core.prefs
+
+# Suppress Eclipse compiler false violation. See https://bugs.eclipse.org/bugs/show_bug.cgi?id=353394
+sed -i -e "s/reportMethodCanBeStatic=error/reportMethodCanBeStatic=warning/g" org.eclipse.jdt.core.prefs
```

Comment:
Eclipse compiler false reports [this method](https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/master/sevntu-checks/src/test/java/com/github/sevntu/checkstyle/internal/ChecksTest.java#L56) as "may be static".